coral/src/app/components/ComplexNativeSelect.tsx

```diff
@@ -0,0 +1,73 @@
+import { NativeSelect } from "@aivenio/aquarium";
+import { ChangeEvent } from "react";
+
+type ComplexNativeSelectProps<T> = {
```

Comment:
We might be able to keep the type simpler, if we would extend either `React.SelectHTMLAttributes<HTMLSelectElement>` or `NativeSelectProps `. It would also logically make sense, as this aims to extend the `<NativeSelect />` with some functionality.

Reply:
Sounds good! I'll look into it!